### PR TITLE
problemList: add missing windows-aarch64 exclusions

### PR DESCRIPTION
**Pull request by gdams**
most of these are defined in https://github.com/adoptium/aqa-tests/issues/4155#issuecomment-1920132183

Grinder: https://ci.adoptium.net/job/Grinder/11258/

### CODE REVIEW

**Review by llxia — APPROVED** [review]

File: openjdk/excludes/ProblemList_openjdk21.txt [hunk 1]
```diff
@@ -382,6 +385,10 @@ compiler/codegen/aes/TestAESMain.java https://github.com/adoptium/aqa-tests/issu
 
 # jdk_jfr
 
+jdk/jfr/event/runtime/TestDumpReason.java https://github.com/adoptium/aqa-tests/issues/4155
+jdk/jfr/event/runtime/TestShutdownEvent.java https://github.com/adoptium/aqa-tests/issues/4155
+jdk/jfr/jvm/TestDumpOnCrash.java https://github.com/adoptium/aqa-tests/issues/4155
```
Comment (llxia) [inline openjdk/excludes/ProblemList_openjdk21.txt:390]: There is no platform specified

Comment (gdams) [reply openjdk/excludes/ProblemList_openjdk21.txt:390]: updated PTAL, I wasn't sure if I should remove these excludes from the Microsoft vendor problemList or not?

Comment (llxia) [reply openjdk/excludes/ProblemList_openjdk21.txt]: These excludes should be removed from the Microsoft vendor problemList. There is no need to duplicate them. openjdk/excludes/ProblemList_openjdkxxx.txt applies to JDK_IMPL=hotspot (regardless of vendor).

**Review by llxia — APPROVED** [review]

File: openjdk/excludes/ProblemList_openjdk21.txt (hunk 1 above)
Comment (gdams) [reply openjdk/excludes/ProblemList_openjdk21.txt]: I've removed the duplicated excludes from the Msft vendor folder

**Review by smlambert — APPROVED** [review]

**Pull request merged**